### PR TITLE
topic/dictionary functions

### PR DESCRIPTION
**Pull request by jgoetzinger**
In many testcases it is desired to convert the output of a command into a dictionary. The function "split_output_to_dict" executes a command with a given command protocol and generates a dictionary from the output.

### CODE REVIEW

File: labgridhelper/dictionary_functions.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def split_output_to_dict(command_protocoll, command, delimiter="=", strip_chars=""):
```
Comment (Emantor) [inline labgridhelper/dictionary_functions.py]: `command_protocol` with one `l` please. Other occurences in the file need to be fixed as well.

**Review by Emantor — COMMENTED** [review]
Change looks good, however I think the submodule `dictionary_functions` is quite long. Maybe `labgridhelper.dict`? Or reference it as a module for the command protocol, i.e. `labgridhelper.command`?
I think the helper function should not execute the command and leave that to the caller. This affords a more flexible use which does not rely on using the `run_check` call and allows use with `run` instead. We also only need to take a command_protocol output in this case.
Than it should also be renamed to `split_command_output_to_dict`.

**Comment by jgoetzinger** [comment]
Thanks for the feedback. I changed the function so that it no longer executes the command. Instead it takes the command output as an input argument. The module and function names are shorter now.

**Review by Emantor — APPROVED** [review]

**Comment by Emantor** [comment]
Thanks!

**Pull request merged**